taxi_service/settings.py

```diff
@@ -44,6 +44,8 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "crispy_forms",
+    "crispy_bootstrap4",
+    "crispy_forms",
```

Comment:
There is a duplicate entry for 'crispy_forms' in the INSTALLED_APPS list. Consider removing one of these entries to avoid redundancy.